modules/Users/views/view.edit.php

```diff
@@ -219,6 +219,22 @@ public function display()
                     <input type="button" id="SAVE_HEADER" title="{$APP['LBL_SAVE_BUTTON_TITLE']}" accessKey="{$APP['LBL_SAVE_BUTTON_KEY']}"
                           class="button primary" onclick="var _form = $('#EditView')[0]; if (!set_password(_form,newrules('{$minpwdlength}','{$maxpwdlength}','{$REGEX}'))) return false; if (!Admin_check()) return false; _form.action.value='Save'; {$CHOOSER_SCRIPT} {$REASSIGN_JS} if(verify_data(EditView)) _form.submit();"
                           name="button" value="{$APP['LBL_SAVE_BUTTON_LABEL']}">
+EOD
```

Comment:
Why are we not amending the original $action_button_header? It pretty much looks identical, what is the difference?

Reply:
Each $action_button_header[] element is for a different button (Save, cancel, etc). This just adds a script to the page to handle submitting through the keyboard without clicking a button.